.github/workflows/publish.yml

```diff
@@ -42,22 +41,40 @@ jobs:
       - name: Update app version in package.json
         run: node .github/workflows/publish/update-package-json.cjs ${{ needs.release.outputs.new_release_version }}
 
-      - name: Set up QEMU
-        uses: docker/setup-qemu-action@v3
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20
+      - run: npm ci
+      - run: npm run build
 
-      - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v3
+      - name: Deploy
+        uses: cloudflare/wrangler-action@v3
+        with:
+          apiToken: ${{ secrets.CF_API_TOKEN }}
+          accountId: ${{ secrets.CF_ACCOUNT_ID }}
+          command: pages publish .svelte-kit/cloudflare --project-name hollama
+        
+  docker:
+    needs: release
+    if: ${{ needs.release.outputs.new_release_published == 'true' }}
 
-      - name: Login to GitHub Container Registry
-        uses: docker/login-action@v3
+    name: Docker image
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Update app version in package.json
+        run: node .github/workflows/publish/update-package-json.cjs ${{ needs.release.outputs.new_release_version }}
```

Comment:
Even though `semantic-release` does automatically update `package.json` with the latest version, we want `semantic-release` to remain as a stand-alone job so we don't have to re-run it in case the build jobs fail.

Therefore we need to manually update the version in `package.json` for the `cloudflare` and `docker` jobs, based on the output of the `release` job.